contracts/RecoveryQuorumFactory.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity ^0.4.8;
+import "./RecoveryQuorum.sol";
+import "./RecoverableController.sol";
+
+contract RecoveryQuorumFactory {
+    event RecoveryQuorumCreated(address recoveryQuorum);
+
+    function CreateRecoveryQuorum(address _controller, address[] delegates) {
+        RecoveryQuorum recoveryQuorum = new RecoveryQuorum(_controller, delegates);
+        RecoverableController controller = RecoverableController(_controller);
```

Comment:
Line 10 and 11 won't work since it doesn't have the right to do this yet. So just instantiate the contract and create the event. We have to do the rest with a separate transaction from mobile.

Also add tests please. Look at tests/identityFactory.js for example of how to do it.